examples/tox21/tox21_dataset_exploration.ipynb

```diff
@@ -174,11 +174,11 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 11764/11764 [00:23<00:00, 501.06it/s]\n",
+      "100%|███████████████████████████████████████████████████████████████████████████| 11764/11764 [00:29<00:00, 401.74it/s]\n",
       "INFO:chainer_chemistry.dataset.parsers.sdf_file_parser:Preprocess finished. FAIL 0, SUCCESS 11757, TOTAL 11757\n",
```

Comment:
RDKit version also affects this result. In newer version, `SUCCESS 11757` becomes `SUCCESS 11758`. Therefore, the note about RDKit version should come before here.